frontend/src/bundles/studio/components/timeline/rows/scenes-row.tsx

```diff
@@ -25,14 +35,52 @@ const ScenesRow: React.FC = () => {
         pixelsToValue(buttonWidthInPixels),
     );
 
+    const dispatch = useAppDispatch();
+
+    const handleAddClick = useCallback(() => {
+        dispatch(studioActions.addScene());
+        dispatch(studioActions.setMenuActiveItem(MenuItems.AVATARS));
+    }, [dispatch]);
+
+    const handleDeleteClick = useCallback(
+        (event: React.MouseEvent<HTMLButtonElement>) => {
+            event.stopPropagation();
+            const id = event.currentTarget.dataset['id'];
+            if (id) {
+                dispatch(studioActions.deleteScene(id));
+            }
+        },
+        [dispatch],
+    );
```

Comment:
Let's add restriction, so that the user cannot delete the scene if it is the only one remaining

Reply:
Great suggestion, thanks, I'll implement it.